### PR TITLE
Enhancing Documentation - initial commit.

### PR DESCRIPTION
**Pull request by palani-sayari**
# Updates to examples and README files

# Description
This pull request includes updates to the example scripts and documentation:
- **Examples Folder**: Modifications to example scripts for consistency and improved error handling.
- **README.md**: Updated to reference example scripts with links, reducing duplication.

# Types of Changes

- [x] Documentation update (non-breaking change)
- [x] New feature in examples (non-breaking enhancement) - include Sayari-error code and logging

# Checklist
- [x] My code adheres to the project's style and conventions.
- [x] Documentation has been updated to reflect changes.
- [x] Tested example scripts to ensure functionality.

---

### CODE REVIEW

File: package.json
Comment (palani-sayari) [inline package.json:1]: These changes were done when did the npm installation steps.  Please feel free to mark the changes as unnecessary.

File: yarn.lock
Comment (palani-sayari) [inline yarn.lock:1]: The following changes occurred during the npm install process. Please feel free to disregard them if they are deemed unnecessary.

File: examples/config.mjs
Comment (palani-sayari) [inline examples/config.mjs:1]: The code is largely self-explanatory, though it currently has minimal inline comments. Please let me know if additional inline documentation would be beneficial.

File: README.md
Comment (palani-sayari) [inline README.md:1]: I’m blaming my VSCode for the changes here—it’s just too good at tidying up whenever I save the file! 😅

File: yarn.lock
Comment (davidkonigsberg) [reply yarn.lock:1]: These files are auto-generated by the fern generated and should not be modified manually.

File: package.json
Comment (davidkonigsberg) [reply package.json:1]: These files are auto-generated by the fern generated and should not be modified manually.

File: package-lock.json
Comment (davidkonigsberg) [inline package-lock.json:2]: These files are auto-generated by the fern generated and should not be modified manually.

File: README.md
Comment (davidkonigsberg) [reply README.md:1]: No worries, all the changes look good to me anyways :)

File: examples/config.mjs
Comment (davidkonigsberg) [inline examples/config.mjs:17]: We may want to only trigger this 'if/else' logic if the ENV vars are not already set. I wouldn't want to trigger this warning if a user has elected to export the vars directly instead of using an env file (my apologies if that isn't possible here)

File: examples/hello-world.mjs
Comment (davidkonigsberg) [reply examples/hello-world.mjs:30]: I like this for simplicity's sake in the 'hello world' example, but feel free to do something more elegant in the other examples if you would like to.

File: examples/hello-world.mjs
Comment (palani-sayari) [reply examples/hello-world.mjs:49]: Happy to defer to views on this one.

I think this is more structured and more importantly this can be 'drag-dropped' into anyone wanting to use/extend the code. :-)

File: package-lock.json
Comment (palani-sayari) [reply package-lock.json:2]: I shall roll back changes and use the files from main here.

File: package.json
Comment (palani-sayari) [reply package.json:1]: Will roll back the changes here.

File: examples/hello-world.mjs
Comment (davidkonigsberg) [reply examples/hello-world.mjs:49]: Yeah, I don't feel strongly either. I tend to view hello world examples as being the minimal code needed to test an endpoint and not something people should consider a production ready example.... at the same time, it is easy enough to read. I will approve whatever is here when this PR is sent back for re-review :)

File: examples/config.mjs
Comment (palani-sayari) [reply examples/config.mjs:17]: Good spot! 👍

File: yarn.lock
Comment (davidkonigsberg) [inline yarn.lock:12]: this file is also auto-generated, please revert

**Review by davidkonigsberg — APPROVED** [review]

**Pull request merged**
